src/Stratis.Bitcoin.Features.Api/Startup.cs

```diff
@@ -27,7 +27,7 @@ public Startup(IHostingEnvironment env)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            // Add service and create Policy to allow Cross-Origin Requests
+            // Save service and create Policy to allow Cross-Origin Requests
```

Comment:
It looks like that the old comment made better sense, what is the rationale?

Reply:
Sorry this change was not intended, fat fingers replace